packages/api/src/user/user.ts

```diff
@@ -113,13 +114,15 @@ export const getPlatformUserBalance = async (
   call: Call,
   { userType, platformUserId, tokenId }: PlatformUserTokenBalancesArgs,
 ) => {
-  const response = await call<Response<{ token: string }>>({
+  const response = await call<Response<{ token: TokenSymbolType }>>({
     url: `/v1/platforms/${userType}/users/${platformUserId}/balances/${tokenId}`,
     method: 'GET',
     authorization: true,
   })
 
-  return response.data
+  const token = response.data.token.symbol
+
+  return { ...response.data, token }
```

Comment:
We should never modify/re-structure response. Instead, you have to return the original response & show it correctly in the example.

Reply:
Do you suggest changing the formatting of the response  we pass to the `printTable` function in the example instead?